### PR TITLE
fix! Revert "fix(comid): bring ValueTriple into alignment with rev05"

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**
This reverts commit b015c203eea7f5ca4a6aeee0d8fa9d4802ddd3fe.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -323,10 +323,16 @@ github.com/veraison/apiclient v0.3.1-0.20240807160142-9141ad363e45 h1:o+gCzGtusX
 github.com/veraison/apiclient v0.3.1-0.20240807160142-9141ad363e45/go.mod h1:LCXFZ3D/tJ3HLAOHUg8bnAKGvgTl53e1ntwdwjVbQ5A=
 github.com/veraison/corim v1.1.3-0.20240814105452-be7ec4829479 h1:dcKW+Nugh2Cs/ihz6xAmmTfi4v5flaLTg6MiZ8gN3N8=
 github.com/veraison/corim v1.1.3-0.20240814105452-be7ec4829479/go.mod h1:sYmwruIqD5+83OcvMg6WUDTTWq8AWM6QbVQhbE9VFQM=
```
Comment (jraman567) [inline go.sum:325]:
```suggestion
```

File: go.sum [hunk 2]
```diff
@@ -323,10 +323,16 @@ github.com/veraison/apiclient v0.3.1-0.20240807160142-9141ad363e45 h1:o+gCzGtusX
...
+github.com/veraison/corim v1.1.3-0.20241003171039-fe09de9f3764 h1:48GvCJSVsBDaqIiK0m+h4D9rhAwLlzias+F5oLId/Fg=
+github.com/veraison/corim v1.1.3-0.20241003171039-fe09de9f3764/go.mod h1:Wj3a6bSo7+3peVGjwGayHDALILh4PHMngDhgBYUbVLk=
 github.com/veraison/eat v0.0.0-20210331113810-3da8a4dd42ff h1:r6I2eJL/z8dp5flsQIKHMeDjyV6UO8If3MaVBLvTjF4=
 github.com/veraison/eat v0.0.0-20210331113810-3da8a4dd42ff/go.mod h1:+kxt8iuFiVvKRs2VQ1Ho7bbAScXAB/kHFFuP5Biw19I=
 github.com/veraison/go-cose v1.1.1-0.20230825153510-da0f9a62ade7 h1:KcKzBthSrSZIUEWBjVvkuk/DE3PyYFbXZxhx5byGFtc=
 github.com/veraison/go-cose v1.1.1-0.20230825153510-da0f9a62ade7/go.mod h1:t6V8WJzHm1PD5HNsuDjW3KLv577uWb6UTzbZGvdQHD8=
+github.com/veraison/go-cose v1.2.1 h1:Gj4x20D0YP79J2+cK3anjGEMwIkg2xX+TKVVGUXwNAc=
+github.com/veraison/go-cose v1.2.1/go.mod h1:t6V8WJzHm1PD5HNsuDjW3KLv577uWb6UTzbZGvdQHD8=
```
Comment (jraman567) [inline go.sum:333]:
```suggestion
```

File: go.sum (hunk 1 above)
Comment (jraman567) [reply go.sum:325]: Thanks for the changes, @yogeshbdeshpande ! I am adding some nits to remove unused Go package versions.

**Review by jraman567 — CHANGES_REQUESTED** [review]
Clean up unused Go packages.

File: go.sum (hunk 1 above)
Comment (yogeshbdeshpande) [reply go.sum]: @jraman567 Thanks, I have addressed this by running ``go mod tidy

**Review by jraman567 — APPROVED** [review]

**Comment by jraman567** [comment]
@yogeshbdeshpande, Is there any reason you added a separate commit? It looks like the changes I proposed were the same as yours.

**Comment by yogeshbdeshpande** [comment]
> @yogeshbdeshpande, Is there any reason you added a separate commit? It looks like the changes I proposed were the same as yours.

No i just ran `go mod tidy` to confirm that all unwanted stuff is removed or not ? That all..

**Review by thomas-fossati — APPROVED** [review]

**Pull request merged**